darwin.c

```diff
@@ -7,82 +7,8 @@
 #include <mach-o/dyld.h>
 #include "common.h"
 
-int elevate(char *path, char *prompt, char *iconPath)
```

Comment:
Having to extract and keep track of it's binary, more entities to look and understand. To be self-contained would be good for such a tiny tool.